package.json

```diff
@@ -9,13 +9,13 @@
         "url": "git://github.com/Becklyn/auslese.git"
     },
     "version": "1.1.0",
+    "main": "js/index.js",
     "scripts": {
         "build": "node_modules/.bin/tsc --noEmitOnError --noErrorTruncation --listEmittedFiles --pretty --noUnusedLocals",
         "dev": "node_modules/.bin/tsc --noEmitOnError --noErrorTruncation --listEmittedFiles --pretty -w",
         "prepublishOnly": "npm run-script build",
         "test": "ava"
     },
-    "main": "js/index.js",
     "dependencies": {
         "match-sorter": "^3.1.1",
         "mojave": "^4.5.1",
```

Comment:
Do we want to upgrade immediately to Preact's v10 instead of the rc.0?

Reply:
We are compatible with both stable + the rc.
It's up to the application to force the stable version.

Reply:
But yeah.. if somebody isn't installing it by themselves, then we can at least use the stable version. Bumped it.